app/code/Magento/Paypal/Controller/Express/AbstractExpress/PlaceOrder.php

```diff
@@ -160,8 +162,6 @@ private function processException(\Exception $exception, string $message): void
      */
     protected function _processPaypalApiError($exception)
     {
-        $this->paymentFailures->handle((int)$this->_getCheckoutSession()->getQuoteId(), $exception->getMessage());
```

Comment:
Do we really need to remove this call? Could you please provide more details on this solution?

Reply:
Looks like it was removed accidentally while resolving a merge conflict.